File: Programmers/x만큼간격이있는n개의숫자.py
def solution(x, n):
    answer = []
    for i in range(1,n+1):
        answer.append(x*i)
    return answer

print('예상답1 : [2,4,6,8,10], 실제답1 : ' + str(solution(2,5)))
print('예상답2 : [4,8,12], 실제답2 : ' + str(solution(4,3)))
print('예상답3 : [-4,-8], 실제답3 : ' + str(solution(-4,2)))

File: Java/Programmers/x만큼간격이있는n개의숫자.java

public class x만큼간격이있는n개의숫자 {
	public static void main(String args[]) {
		Solutionx만큼간격이있는n개의숫자 s = new Solutionx만큼간격이있는n개의숫자();
		long[] res = s.solution(-4, 2);
		for(int i=0; i<res.length; i++) {
			System.out.print(res[i] + ", ");
		}
	}
}

class Solutionx만큼간격이있는n개의숫자 {
	public long[] solution(long x, int n) {
		long[] answer = new long[n];
		for(int i=0; i<n; i++) {
			answer[i] = x*(i+1);
		}
		return answer;
	}
}
